Sources/XcodeGenKit/PathKitExtension.swift

```diff
@@ -0,0 +1,17 @@
+import PathKit
+
+extension Path {
+
+    /// Treat this as a resource instead of a normal directory.
+    ///
+    /// - NOTE: "lproj" is handled differently.
+    var isFileLikeDirectory: Bool {
+        guard let ex = self.extension else { return false }
+        return ["imageset",
+                "xcassets",
+                "complicationset",
+                "appiconset",
+                "xcdatamodeld"].contains(ex)
```

Comment:
We should add "bundle" and "framework" to this as well.

Reply:
Will do👌

Reply:
I wonder if there is an API to detect this? It seems to be OS defined.

Reply:
Found an API to get UTI in string. 1e21b1d
It's `"public.folder"` if it's a normal folder.
`lproj` appears to be `"public.folder"` too. It looks problematic, but we're whitelisting `lproj`, so it wouldn't be a problem.